wiremock-docker/__files/basm-api/locations/MULTIPLE.json

```diff
@@ -70,7 +70,7 @@
         "latitude": null,
         "longitude": null,
         "disabled_at": null,
-        "created_at": "{{now format='yyyy-MM-dd'}}"
+        "created_at": "{{now}}"
```

Comment:
Do these want removing?

Reply:
Left in attempt to be consistent with the response.  Though saying that the same should apply to all the other JSON files really...